comp/forwarder/defaultforwarder/default_forwarder.go

```diff
@@ -443,6 +457,15 @@ func (f *DefaultForwarder) State() uint32 {
 	return f.internalState.Load()
 }
 
+// domainAPIKeyMap used by tests to get API keys from each domain resolver
+func (f *DefaultForwarder) domainAPIKeyMap() map[string][]string {
+	apiKeyMap := map[string][]string{}
+	for domain, dr := range f.domainResolvers {
+		apiKeyMap[domain] = dr.GetAPIKeys()
+	}
+	return apiKeyMap
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏If this is only used by tests it should likely move to a test file

Reply:
Ok, done